src/storage/conversion/MarkdownToHtmlConverter.ts

```diff
@@ -4,16 +4,10 @@ import type { Representation } from '../../http/representation/Representation';
 import { TEXT_HTML, TEXT_MARKDOWN } from '../../util/ContentTypes';
 import { readableToString } from '../../util/StreamUtil';
 import type { TemplateEngine } from '../../util/templates/TemplateEngine';
+import { BaseTypedRepresentationConverter } from './BaseTypedRepresentationConverter';
 import type { RepresentationConverterArgs } from './RepresentationConverter';
-import { TypedRepresentationConverter } from './TypedRepresentationConverter';
 
-/**
- * Converts Markdown data to HTML.
- * The generated HTML will be injected into the given template using the parameter `htmlBody`.
- * A standard Markdown string will be converted to a <p> tag, so html and body tags should be part of the template.
- * In case the Markdown body starts with a header (#), that value will also be used as `title` parameter.
- */
-export class MarkdownToHtmlConverter extends TypedRepresentationConverter {
```

Comment:
Why the comment?

Reply:
Because the linter removes all the comments above a class if it sees it can remove the lowest import in the list of imports. :P Thanks for catching this.

Reply:
Actually all converters lost their comment, would have been quite bad if this wasn't spotted 😅